util/fipstools/delocate/delocate.peg.go

```diff
@@ -1,6 +1,6 @@
 package main
 
-// Code generated by /home/ubuntu/go/bin/peg delocate.peg DO NOT EDIT.
+// Code generated by /Users/jakemas/go/bin/peg /Users/jakemas/Documents/PQ/ML-DSA/jan6/aws-lc/util/fipstools/delocate/delocate.peg DO NOT EDIT.
```

Comment:
not major, but may want to revert this line.

Reply:
Thanks! [49921f0](https://github.com/aws/aws-lc/pull/2096/commits/49921f0753147eda549a4d0e3e7ce71264bf2ad9). Making a note to point out in the [documentation ](https://github.com/aws/aws-lc/blob/73f0497dcc91a24cdf5a29b756c5fd545eeb44cc/util/fipstools/delocate/delocate.peg#L19-L24) that you need to change this line after!